src/styles/review.css

```diff
@@ -0,0 +1,8 @@
+.review__stars {
+  display: flex;
+}
+
+.review__count {
+  padding-right: 16px;
+  font-size: 10px;
+}
```

Comment:
a file without the main block? interesting...

Reply:
I thought you couldn't put `display: flex` on blocks so I avoided it, which resulted in no main block here, but Paul already cleared that up for me, fixed